RevenueCatUI/Data/TemplateViewConfiguration.swift

```diff
@@ -236,54 +236,56 @@ extension TemplateViewConfiguration.PackageConfiguration {
             }
 
         case let .multiTier(tiers, localization):
-            let allTiers: [PaywallData.Tier: (package: MultiPackage, tierName: String)] = try Dictionary(
-                tiers.compactMap { tier in
-                    guard let localization = localization[tier.id] else {
-                        throw TemplateError.missingLocalization(tier)
-                    }
-
-                    let tierName = localization.tierName ?? ""
-
-                    let filteredPackages = Self.processPackages(
-                        from: packages,
-                        filter: tier.packages,
-                        activelySubscribedProductIdentifiers: activelySubscribedProductIdentifiers,
-                        localization: localization,
-                        locale: locale,
-                        showZeroDecimalPlacePrices: showZeroDecimalPlacePrices
-                    )
+            let filteredTiers: [(PaywallData.Tier, (MultiPackage, String))] = try tiers.compactMap { tier in
```

Comment:
I think it's helpful to maintain the labels on the tuple, `(package: MultiPackage, tierName: String)` rather than `(MultiPackage, String)`.

Reply:
I agree! https://github.com/RevenueCat/purchases-ios/pull/4243/commits/24c1f689bcf55970b0381420bef0e831ea80a156